web/views.py

```
@@ -4150,6 +4150,11 @@ def meme_list(request):
     return render(request, "memes.html", {"memes": page_obj, "subjects": subjects, "selected_subject": subject_filter})
 
 
+def meme_detail(request, slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `meme_detail`

(ANN201)